src/plugins/foxPage/hooks/useOtherOpportunities.ts

```diff
@@ -34,14 +34,24 @@ export const useOtherOpportunities = (assetId: AssetId) => {
           opportunities: [
             {
               title: 'ETH-FOX UNI V2 Farm',
-              isLoaded: isFarmingAprLoaded,
-              apy: isFarmingAprLoaded ? farmingApr : null,
+              isLoaded: isFarmingAprV2Loaded,
+              apy: isFarmingAprV2Loaded ? farmingAprV2 : null,
               link: 'https://fox.shapeshift.com/fox-farming/liquidity/0x470e8de2ebaef52014a47cb5e6af86884947f08c/staking/0x212ebf9FD3c10F371557b08E993eAaB385c3932b/get-started',
               icons: [
                 'https://assets.coincap.io/assets/icons/eth@2x.png',
                 'https://assets.coincap.io/assets/icons/fox@2x.png',
               ],
             },
+            {
+              title: 'ETH-FOX UNI V4 Farm',
+              isLoaded: isFarmingAprV4Loaded,
+              apy: isFarmingAprV4Loaded ? farmingAprV4 : null,
+              link: 'https://fox.shapeshift.com/fox-farming/liquidity/0x470e8de2ebaef52014a47cb5e6af86884947f08c/staking/0x24fd7fb95dc742e23dc3829d3e656feeb5f67fa0',
```

Comment:
should this also go to `/get-started`?

Reply:
Nice catch ser! Updated in 92a6de37093ea2d5dd83068398ed74ee9168c551 and also updated the issue spec to match it!